README.md

```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
+     
+      `composer install` 
```

Comment:
For code lines, if you indent with four spaces, you won't need the backticks for marking the text as code.

Please see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code for a Markdown cheat sheet.